src/openzaak/components/catalogi/api/serializers/informatieobjecttype.py

```diff
@@ -31,6 +56,10 @@ class Meta:
             "begin_geldigheid",
             "einde_geldigheid",
             "concept",
+            "besluittypen",
+            "informatieobjectcategorie",
+            "trefwoord",
+            "omschrijving_generiek",
```

Comment:
I'm missing `zaaktypen` I think?

![image](https://github.com/open-zaak/open-zaak/assets/96970/692ebc3b-6aaa-4d81-a444-27b716dad709)

Reply:
Thanks for catching it! It looks like I've missed it. I'll add `zaaktypen`

Reply:
`InformatieObjectType.zaaktypen` property is added